Sources_v3/Database/DTOs/ChannelDTO.swift

```diff
@@ -155,6 +155,11 @@ extension ChannelDTO {
     }
 }
 
+extension ChannelDTO {
+    /// Snapshots the current state of `ChannelDTO` and returns an immutable model object from it.
+    func asModel<ExtraData: ExtraDataTypes>() -> ChannelModel<ExtraData> { .create(fromDTO: self) }
```

Comment:
Ideally, you can also make `create(fromDTO:)` fileprivate

Reply:
fixed in aed02f9263fc5863b5ae627e5303ffcfb94e08bd and 1516789ea2be07b775f0133c68279c01d437a333